resources/i18n/de/strings.xml

```diff
@@ -1,6 +1,26 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools" xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2" tools:ignore="MissingTranslation">
 
+  <!-- XHED: Admission Scenario - Card Title -->
+  <string name="ADMISSION_SCENARIO_LABEL_TEXT">"Status für folgendes Bundesland"</string>
+  <!-- XTXT: Admission Scenario - Card Title -->
```

Comment:
```suggestion
  <!-- XTXT: Admission Scenario - Card Subtitle Text -->
```

Reply:
Resolved in https://github.com/corona-warn-app/cwa-app-ccl/commit/e6618f359606a912fdaead0db0dc1ea5c994f513